UnzipKit.xcodeproj/project.pbxproj

```diff
@@ -447,6 +450,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```

Comment:
Please revert this line (you must have missed it when you backed out the rest of your project changes).

Reply:
This is because Xcode is automatically generated, I try to recover, but there is still a problem. Maybe you can help me

Reply:
I'm sure Xcode is putting this here without your direct action. I'm just saying not to commit it to the repo. If you're using Xcode for your commits, you can uncheck this change. You can also revert it back outside of Xcode and commit.